nucypher/cli/painting.py

```diff
@@ -447,6 +448,57 @@ def paint_staking_confirmation(emitter, staker, new_stake):
     emitter.echo(next_steps, color='green')
 
 
+def paint_stakes(emitter, stakeholder, paint_inactive: bool = False, staker_address: str = None):
+    headers = ('Idx', 'Value', 'Remaining', 'Enactment')
```

Comment:
last period will be helpful here, because it's hard to calculate this value from provided information

Reply:
:heavy_check_mark: